redisinsight/values.yaml

```diff
@@ -0,0 +1,115 @@
+# Default values for redisinsight.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  # -- The repository of the image
+  repository: redis/redisinsight
+  # -- The pullPolicy used when pulling the image
+  pullPolicy: IfNotPresent
+  # -- The tag of the iamge. Overrides the image tag whose default is the chart appVersion.
+  tag: "2.48.0"
+
+server:
+  # -- The number of pods to be deployed for server
+  replicas: 1
+  # -- Specific values for the service resource
+  service:
+    # -- Service labels
+    labels: {}
+    # -- Service annotations
+    annotations: {}
+  
+  # -- Collection of ports
+  ports:
+    # -- Port through which the UI is exposed
+    ui: 5540
+
+  # -- Extra environment variables to inject
+  extraEnvVars: {}
+
+# -- Ingress related values
+ingress:
+  # -- Whether to enable the ingress or not
+  enabled: false
+  # -- IngressClass used for the ingress
+  className: ""
+  # -- Ingress annotations
+  annotations: {}
+  hosts:
+    - host: redisinsight.local
+      paths:
+        - path: /
```

Comment:
**suggestion:** To access the webapp, we need to set this to `/*`. It would be nice to set it as the default.

```suggestion
        - path: /*
```